frontend/ui/elements/screensaver_menu.lua

```diff
@@ -137,6 +138,15 @@ return {
                 checked_func = whiteBackground,
                 callback = function()
                     G_reader_settings:saveSetting("screensaver_white_background", not whiteBackground())
+                    G_reader_settings:flipFalse("screensaver_no_background")
+                end,
+            },
+            {
+                text = _("Leave background as-is behind message and images"),
```

Comment:
What does that mean precisely? Alpha-channel blending might have weird results or something?

Reply:
It just means that alpha-blending will actually be useful, since we won't be clearing to white or black, or drawing any kind of background behind the image.

So you can effectively use transparent images and have the transparency stay useful.

Prepare to be amazed by my test image... :D

![zzz](https://user-images.githubusercontent.com/111974/54378654-0bdc6300-4688-11e9-87b8-38603ff8de48.png)

Reply:
Hmm, still something a bit off with alpha-blending, it looks darker than it should (and there's a bit of solarization on the edge of the lighter gray...).

Reply:
i.e., original overlay:
![forma-sleepy](https://user-images.githubusercontent.com/111974/54379302-72ae4c00-4689-11e9-89a4-ade060eca9dc.png)

And essentially the same as earlier but rendered w/ FBInk:
![zzz_fbink](https://user-images.githubusercontent.com/111974/54379334-7e9a0e00-4689-11e9-8879-810f1db71267.png)

Reply:
Note to self: try with a G8A input image, to rule out a grayscaling pass somewhere.

Reply:
After looking at this a bit,

this looks suspiciously like the PNG matte background bleeding through (also visible when both MuPDF & LodePNG flatten the image), so, meh.

Reply:
Couldn't manage to find much info about how to set that matte color (short of old PhotoShop versions), so, let's say it's a weird PNG thing.

Reply:
Haha! Figured it out.

MuPDF stores pixmaps in [premultiplied alpha](https://ghostscript.com/~robin/mupdf_explored.pdf#subsection.10.3.2), while our blending code expects straight alpha.

I'll cobble something together...